lib/notion/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 module Notion
-  VERSION = '1.1.0'
-  NOTION_REQUEST_VERSION = '2022-02-22'
+  VERSION = '1.1.1'
```

Comment:
I’ll cut a new version in a separate commit (so that I can add a changelog entry, etc…), can you revert this change? Thanks!

Reply:
Reverted the commit.